components/nexus-core/src/main/java/org/sonatype/nexus/proxy/storage/remote/httpclient/HttpClientManagerImpl.java

```diff
@@ -55,6 +61,10 @@
     extends ComponentSupport
     implements HttpClientManager
 {
+  private static final String NX_REMOTE_ENABLE_CIRCULAR_REDIRECTIONS_KEY = "nexus.remoteStorage.enableCircularRedirectionsForHosts";
```

Comment:
minor: "nexus.remoteStorage.enableCircularRedirectsForHosts" is more consistent with the method name